ui/guest/src/react/GuestProxy.tsx

```diff
@@ -232,17 +234,17 @@ export function GuestProxy() {
           }
 
           const phyRecord = ElementRegistry.fromICEId(iceId);
-          const $el = $(phyRecord.element);
-          const $targetSibling = $el.parent().children().eq(targetIndexParsed);
+          const el = phyRecord.element;
+          const targetSibling = el.parentElement.children[targetIndexParsed];
 
           // Move...
           if (currentIndexParsed < targetIndexParsed) {
-            $el.insertAfter($targetSibling);
+            el.parentElement.insertBefore(el, targetSibling.nextSibling);
```

Comment:
Make sure this doesn't result in out of bounds or null when using .nextSibling on target element. Could the target element be the last child?

Reply:
Same as https://github.com/craftercms/studio-ui/pull/3356#discussion_r1240436667